### DISCUSSION TITLE
Fail to install Blender plugin for CC3

### DISCUSSION THREAD

**Discussion by Pizhlo** [soupday/cc3_blender_tools#73] [Q&A]
I followed instructions of installation step-by-step, but when I launched Python script in CC3, I got this message in the Console Log: "
<class 'AttributeError'>,  File: C:/Program Files/Reallusion/Character Creator 3/Bin64/OpenPlugin/CC3-Blender-Tools-Plugin-1_0_4/main.py,  Line: 75"

What should I do to solve it?

**Comment by soupday** [comment]
Is that from the **Plugins > Blender Autosetup > Import From Blender** menu?

It shouldn't be crashing there at all, unless the python API in CC3 isn't fully installed properly. Which is possible, some people couldn't get the plugin to work until re-installing CC3.

**Reply by Pizhlo** [reply]
This is from Script > Load Python menu. I even don't have the 'Blender Autosetup' button.
So what should I do? Reinstall CC3?

**Reply by soupday** [reply]
Something it missing from the CC3 installation then. Re-installing might fix it.

**Reply by Pizhlo** [reply]
Thank you for your prompt answer!
